hashicorp-vault-dynamic-secret-ysql-plugin/run-app.sh

```diff
@@ -17,4 +19,31 @@ echo "Running tests"
 
 go clean -testcache
 
-go test -v
\ No newline at end of file
+go test -v 2>&1 | tee vault-ysql-plugin-tests.log
+
+echo "[" >> temp_report.json
+
+if grep "FAIL:" "vault-ysql-plugin-tests.log"; then
+  # Get the lines with 'FAIL'
+  grep -B 1 "FAIL:" vault-ysql-plugin-tests.log > stack4json.log
+  # test_name=`sed -n '/^.*FAIL:\s\+\(\w\+\).*$/s//\1/p' gocql-tests.log`
+  python $WORKSPACE/integrations/utils/create_json.py --test_name "NA" --script_name "vault-ysql-plugin-test" --result FAILED --file_path stack4json.log >> temp_report.json
```

Comment:
Change the script name and test name, refer to liquibase or flyway:
```suggestion
  python $WORKSPACE/integrations/utils/create_json.py --test_name "vault-ysql-plugin-test" --script_name "hashicorp-vault-dynamic-secret-ysql-plugin/start.sh" --result FAILED --file_path stack4json.log >> temp_report.json
```